src/mainboard/starlabs/labtop/variants/kbl/romstage.c

```diff
@@ -4,7 +4,7 @@
 #include <console/console.h>
 #include <soc/romstage.h>
 #include <spd_bin.h>
-#include "spd/spd.h"
+#include "../../spd/spd.h"
```

Comment:
Better to add a line to the Makefile.inc to add the correct path with a -I option rather than using relative paths in the include statements.  Will be something like:

`CPPFLAGS_common += -I$(src)/mainboard/$(MAINBOARDDIR)`

This then makes the include statement:

`#include "spd/spd.h"`